src/app/clusters/basic/basic.c

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
Should we call this dir basic-server, to make it clear that's what lives in here?

Reply:
I'm using the names from SiLabs for now, that work?

Reply:
I can live with it, but do think it's worth being clear about which bits are just server and which parts are just client.

Most importantly, since at least some of our build system bits just build all the files in a given dir, we should make sure the server and client files are in different directories...

Reply:
I found a way to suppress individual objects in the components.mk file, so this is not actually an issue.

Reply:
Alright, thanks!  Might still be nice to make it very clear to people how to only pull int the client or only the server when both are available but that can definitely happen as followups.